hera_sim/utils.py

```diff
@@ -375,10 +375,14 @@ def Jy2T(freqs, omega_p):
     # get actual values of omega_p if it's an interpolation object
     if callable(omega_p):
         omega_p = omega_p(freqs)
-    wavelengths = const.c.value / freqs
-    # scaling went from 1e-23 -> 1e-34 in converting to SI
-    # what is the point of this multiplicative constant?
-    return 1e-34 * wavelengths ** 2 / (2 * const.k_B.value * omega_p)
+    wavelengths = const.c.value / (freqs * 1e9)  # meters
```

Comment:
Are we sure it's wise to start changing the units of frequencies here?

Reply:
I'm not sure I understand what you mean? The old calculation used constants from `aipy`, where everything was in cgs; since `astropy` uses mks by default, I switched this calculation to use mks units when I switched from `aipy` to `astropy`.

Reply:
Hmmm, it looked like now `freqs` had to be passed into the function with different units. That's fine, but will break code. That's also OK since this is v1, but we should definitely make this clear in CHANGELOG etc.

Reply:
Noted.